composer.json

```diff
@@ -37,26 +37,26 @@
     "require": {
         "php": "~8.0.0 || ~8.1.0 || ~8.2.0",
         "dflydev/fig-cookies": "^2.0.1 || ^3.0",
-        "laminas/laminas-cli": "^0.1.5 || ^1.0",
-        "laminas/laminas-diactoros": "^2.11.2",
-        "laminas/laminas-httphandlerrunner": "^2.1",
-        "mezzio/mezzio": "^3.0.2",
-        "psr/container": "^1.0 || ^2.0",
+        "laminas/laminas-cli": "^0.1.5 || ^1.8",
+        "laminas/laminas-diactoros": "^2.11.2 || ^3.0",
```

Comment:
bump diactoros v2 to latest. It had security release.

Reply:
done